LibreHardwareMonitorLib/Hardware/Cpu/CpuLoad.cs

```diff
@@ -42,32 +44,92 @@ private static bool GetTimes(out long[] idle, out long[] total)
         idle = null;
         total = null;
 
-        //Query processor idle information
-        Interop.NtDll.SYSTEM_PROCESSOR_IDLE_INFORMATION[] idleInformation = new Interop.NtDll.SYSTEM_PROCESSOR_IDLE_INFORMATION[64];
-        int idleSize = Marshal.SizeOf(typeof(Interop.NtDll.SYSTEM_PROCESSOR_IDLE_INFORMATION));
-        if (Interop.NtDll.NtQuerySystemInformation(Interop.NtDll.SYSTEM_INFORMATION_CLASS.SystemProcessorIdleInformation, idleInformation, idleInformation.Length * idleSize, out int idleReturn) != 0)
+        if (!Software.OperatingSystem.IsUnix)
```

Comment:
Can you split this code up in two functions, e.g. GetUnixTimes and GetWindowsTimes.